k8s/k8s.bzl

```diff
@@ -51,7 +51,7 @@ def k8s_repositories():
         http_archive,
         name = "com_github_yaml_pyyaml",
         build_file_content = _com_github_yaml_pyyaml_build_file,
-        sha256 = "e9df8412ddabc9c21b4437ee138875b95ebb32c25f07f962439e16005152e00e",
+        sha256 = "ba59d7e97eb131d8f8f52d19cb124bb67772f4c7f4d14cb2919deb885ef8c572",
```

Comment:
Any insight into why/when the sha changed?

Reply:
Seems like it has been fixed. Will revert. [This](https://twitter.com/tgummerer/status/1488493440103030787?s=20&t=DaupVstH5BiSJd0BgLSa_Q) has the info. There is chat in the [issue](https://github.com/bazelbuild/rules_k8s/issues/681) as well